### DISCUSSION TITLE
WooCommerce using standard templates

### DISCUSSION THREAD

**Discussion by finalwebsites** [bootscore/bootscore#900] [Q&A]
Hi,
right now I can change the a category archive page only with hooks which are available in the function **woocommerce_content()**.
Instead I would like to use the templates from WooCommerce, like archive-product.php.
If I want to do that you need to remove the woocommerce.php file. Is there another option that I've missed?

**Comment by crftwrk** [comment]
Hi,

yes, I tried many times to integrate WC into the theme with hooks https://developer.woocommerce.com/docs/classic-theme-development-handbook/ instead the woocommerce.php. But I had no acceptable success here, especially for the breadcrumb. If you have any idea how to solve this, we can remove woocommerce.php. Any PR is highly appreciated.

**Reply by finalwebsites** [reply]
I can't follow you :) In my opinion is the integration using the woocommerce.php file some kind of light integration.
If you don't use it all the WOO template files are used instead. I never used the woocommerce.php file during the last 10 years.

> But I had no acceptable success here, especially for the breadcrumb.
I do this with the header.php file in most of the cases.

**Reply by crftwrk** [reply]
> I can't follow you :) In my opinion is the integration using the woocommerce.php file some kind of light integration.

Please submit a PR, removing the woocommerce.php will be a great thing.

**Reply by finalwebsites** [reply]
Actually, if you remove the woocommerce.php file, the default template function is restored.
Maybe it's an idea to keep the woocommerce.php file inside the child theme?
Let me work on this, I will check what I can share afterwards :)

**Reply by crftwrk** [reply]
Please have a look at https://github.com/bootscore/bootscore/pull/1073
